apps/web/.env.example

```
@@ -30,3 +30,18 @@ VITE_NOVU_APP_ID="your-application-identifier"
 # EU region only — omit for the default US region.
 VITE_NOVU_BACKEND_URL="https://eu.api.novu.co"
 VITE_NOVU_SOCKET_URL="https://eu.ws.novu.co"
+
+# Sentry — client (runtime). Leave VITE_SENTRY_DSN unset or empty to disable Sentry.
+VITE_SENTRY_DSN="https://...@sentry.io/..."
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)